frontend/src/components/user/UserControl.tsx

```diff
@@ -138,27 +114,9 @@ export const UserControl = () => {
     await refreshUser();
   }, [refreshUser]);
 
-  const [authLoginUrl, setAuthLoginUrl] = useState<string | null>(null);
-
-  useEffect(() => {
-    async function fetchEnv() {
-      const res = await fetch("/api/env");
-      const data = (await res.json()) as { auth_login_url: string };
-      data.auth_login_url
-        ? setAuthLoginUrl(data.auth_login_url)
-        : console.error("could not access auth_login_url");
-    }
-    fetchEnv().catch((error) => console.warn("error fetching api/env", error));
-  }, []);
-
   return (
     <>
-      {!user?.token && (
-        <LoginLink
-          navLoginLinkText={t("nav_link_login")}
-          loginUrl={authLoginUrl || ""}
-        />
-      )}
+      {!user?.token && <LoginModal navLoginLinkText={t("nav_link_login")} />}
```

Comment:
this works because the LoginModal component includes the modal control button but if you're expecting to see something like <LoginButton> here, it's a little confusing to see <LoginModal> instead. Something for later, but there's probably a refactor that would make this a little easier to understand